### ISSUE TITLE
Missing grunt plugin when using sass

### ISSUE THREAD

**Issue by futjikato** [bezoerb/generator-grunt-symfony#2]
When using sass as CSS compiler `grunt serve` fails with the follwing message:

```
grunt serve
Running "serve" task

Running "sass:all" (sass) task

jit-grunt: Plugin for the "autoprefixer" task not found.
If you have installed the plugin already, please setting the static mapping.
See https://github.com/shootaroo/jit-grunt#static-mappings

Warning: Task "autoprefixer" failed. Use --force to continue.

Aborted due to warnings.
```

**Comment by bezoerb** [comment]
Couldn't produce the issue locally. Can you give some more information about your configuration?

**Comment by futjikato** [comment]
Already fixed with 05b571a799a5484463c7dd714e5e1f30e7e9a52c
